client/src/components/ShardComponents/Login/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import "./style.css"
+class Login extends Component {
+  state = {
+    email: "",
+    password: "",
+    status: "false"
+  }
+
+  handelChange = event => {
+    const value = event.target.value
+    this.setState({ [event.target.name]: value })
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.submit}>
+        <label htmlFor="email"></label>
+        <input
+          placeholder="Email"
+          type="email"
+          id="email"
```

Comment:
u don't need to use id all the time. If you are not using the id for anything, don't add it.

Reply:
She's using it to link to label here so no worries 🐼